cactusbot/commands/magic/alias.py

```diff
@@ -8,7 +8,7 @@ class Alias(Command):
 
     COMMAND = "alias"
 
-    @Command.command()
+    @Command.command(role="moderator")
```

Comment:
[They are integer-based](https://github.com/CactusDev/CactusBot/pull/134/files/5a5e9421a5b472f6fc8db009c7de3d68de6ab991#diff-fa8091e5f9fd54753ebcf602d35099faR6)... but you also have the option to use strings, just for readability/clarity. (They're the preferred method for the decorators.)